config/initializers/avo.rb

```diff
@@ -0,0 +1,109 @@
+# For more information regarding these settings check out our docs https://docs.avohq.io
+Avo.configure do |config|
+  ## == Routing ==
+  config.root_path = "/avo"
```

Comment:
I think it would be nice to move this into the `/developers/` path like we did with Sidekiq.

```suggestion
  config.root_path = "/developers/avo"
```